Begin2019 (#127)
ilios/iliosproject.org: _posts/2019-01-25-updates.markdown

---
layout: post
title: Welcome to the New Way
date: 2019-01-27  08:00:00
categories: updates
---
__STATUS__:
- current version: __3.60.0__
- next scheduled release: __2/13/2019__

> Welcome to a new year, a new release, and a new set of development goals for the Ilios Project in the coming year! Key features identified as target milestones for our 2019-20 roadmap include:

Reviewer: Starting the sentence with "Welcome to a new year" and ending it with "in the coming year!" is probably worth tweaking too:

```suggestion
Welcome to a new year, a new release, and a new set of development goals for the Ilios Project! Key features identified as target milestones for our 2019-20 roadmap include:
```

*SEARCH*: We have begun a year-long initiative to begin the incorporation of deep search. We expect to include search across the entire curriculum and its content.

> *Session Linking*: Session linking tools will be available in the next release.

Reviewer: via @dartajax filtered though HemingwayApp.com:

```suggestion
*Session Linking*: The next release will allow linking ILMs to upcoming sessions. More session linking capabilities will be available in the future.
```

Reviewer 2: 👍

Reviewer 2: I like @Trott 's change here. I apologize if I `resolved` the conversation. I think this change should be included not "resolved".

---
layout: post
title: Welcome to the New Way
date: 2019-01-23  08:00:00
categories: updates
---
__STATUS__:
- current version: __3.60.0__
- next scheduled release: __2/13/2019__

> Welcome to a new year, a new release, and a new set of development goals for the Ilios Project in the coming year! Among the key features identified as target milestones for our 2019-20 roadmap we are happy to note:

Reviewer: ```suggestion
Welcome to a new year, a new release, and a new set of development goals for the Ilios Project in the coming year! Key features identified as target milestones for our 2019-20 roadmap include:
```

> *SEARCH*: We have begun a year-long initiative to begin the incorporation of deep search to the Ilios platform. We hope to be able to include search across the entire curriculum and its content  - from learning objectives to learning materials - and will keep you all apprised of progress in this area.

Reviewer: ```suggestion
*SEARCH*: We have begun a year-long initiative to begin the incorporation of deep search. We expect to include search across the entire curriculum and its content.
```

> *Session Linking*: Beginning with the next release, the session linking tools will be available for Ilios.

Reviewer: ```suggestion
*Session Linking*: Session linking tools will be available in the next release.
```

> *Configurable options*: We expect to be providing the ability to modify an array of different school level attributes, including the time-window for calendar integrations and dashboard displays, naming conventions, and more.

Reviewer: ```suggestion
*Configurable options*: We plan to provide the ability to change school level attributes. Time-window for calendar integrations and dashboard displays will be configurable options. So will naming conventions, and more!
```

*Selectives*: The ability to tag and group sessions as selectives within the curriculum.

> *Improved and extended rollover*: allowing the rollover of entire group trees, selective choice of attributes for rollover in a course, and better publishing control.

Reviewer: ```suggestion
*Improved and extended rollover*: This will allow the rollover of entire group trees. There will also be selective choice of attributes for rollover in a course. Finally, this will include better publishing control.
```

*Improved Curriculum Inventory tools*: better, more flexible tools to represent your curriculum to the AAMC Curriculum Inventory.

Further information on the roadmap is available in our Github repository.

__HIGHLIGHTS__:

__COMING SOON__:

__MORE DETAILS__:

- [http://bit.ly/IliosWeeklyUpdates](http://bit.ly/IliosWeeklyUpdates){:target="updates"}

> Questions? Comments? Feedback? Please let us know at [support@iliosproject.org](mailto:support@iliosproject.org) or in [https://team-ilios.slack.com/messages/help/](https://team-ilios.slack.com/messages/help/){:target="_slackithelp"}.  (If you have not yet joined our slack channel, you can get started at [https://ilios-slack.herokuapp.com/](https://ilios-slack.herokuapp.com/){:target="_slackit"}.)

Author: nobody likes nits. fixed.

---
layout: post
title: Welcome to the New Way

> date: 2019-01-27  08:00:00

Reviewer: Today is the 28th, no?
